test/integration/web_console_access_test.go

```diff
@@ -194,7 +194,7 @@ func TestAccessOriginWebConsoleMultipleIdentityProviders(t *testing.T) {
 	linkRegexps := make([]string, 0)
 
 	// Verify that the plain /login URI is unavailable when multiple IDPs are in use.
-	urlMap["/login"] = urlResults{http.StatusForbidden, ""}
+	urlMap["/login"] = urlResults{http.StatusNotFound, ""}
```

Comment:
this status code changed because the oauth server no longer falls through to our "normal" chain.  If you request a path handled by the oauth server, it is the end of the chain and will 404 on you if you fall through.